### PR TITLE
Adds pagination

### PR DESCRIPTION
**Pull request by rdonati**

### CODE REVIEW

File: .prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// prettier.config.js or .prettierrc.js
+module.exports = {
+  trailingComma: "es5",
+  tabWidth: 2,
+  semi: true,
+  singleQuote: false,
+};
```
Comment (rdonati) [inline .prettierrc.js:7]: Config file for vscode extension.. not relevant to the PR, but I needed it so that I didn't screw up the stying

**Comment by swjeong0825** [comment]
Yo that is brilliant!

**Review by swjeong0825 — APPROVED** [review]

**Comment by swjeong0825** [comment]
I resolved a merge conf in a way that accepts all incoming changes

File: frontend/src/pages/JobFeed/components/JobPreviewCard.tsx [hunk 2]
```diff
@@ -1,30 +1,32 @@
-import { Paper, Typography } from '@mui/material'
```
Comment (CosineP) [inline frontend/src/pages/JobFeed/components/JobPreviewCard.tsx:1]: Formatting noise again. Potential solution, if you didn't change the file at all, don't commit it here and start to make a plan for project wide formatting migration. If you have changed a file, then okay fine it is what it is. (But a description of the change would help reading thru the noise)

**Review by CosineP — COMMENTED** [review]
One thing i don't know if was specified in the design doc but i would like to see discussed: putting the page in the URL, so if a tab sleeps or a page is bookmarked it can be returned to for further searching. Since people are probably searching for a Particular job rather than any job not necessarily a huge priority, but nice to have and pretty easy with react router

File: frontend/src/pages/JobFeed/components/JobCard.tsx [hunk 3]
```diff
@@ -1,43 +1,57 @@
-import { useState } from 'react'
-import { Button, Paper, Typography } from '@mui/material'
```
[the remark this answers is not in the document]
Comment (swjeong0825) [reply frontend/src/pages/JobFeed/components/JobCard.tsx:2]: I think that is because Ric adds the prettier configuration on this branch.

**Comment by rdonati** [comment]
@CosineP 

**Re formatting:** I agree that we need something project-wide. This was just a bandaid to help avoid even more noise in this PR. I think the super noisy files have one change/file and if I remember correctly it's just removing a margin.

**Re putting the page in the url:** In support of this, but I'm going to leave it out of this PR for now.

**Re empty divs:** I included these to fill empty grid cells without having to use grid-template-areas. I added a commit to change the approach to use grid-template areas

**Review by CosineP — APPROVED** [review]
Alright then, sounds good!

**Comment by CosineP** [comment]
The coercion seems fine. Ultimately, the task is parsing so it's gonna have to happen one way or another.

**Pull request merged**